plugins/woocommerce/client/admin/client/settings-payments/components/offline-payment-gateways.tsx

```diff
@@ -0,0 +1,141 @@
+/**
+ * External dependencies
+ */
+import { Card, CardHeader } from '@wordpress/components';
+import React, { useState } from '@wordpress/element';
+import { decodeEntities } from '@wordpress/html-entities';
+import { __ } from '@wordpress/i18n';
+import { List } from '@woocommerce/components';
+import { useSelect } from '@wordpress/data';
+import { PAYMENT_GATEWAYS_STORE_NAME } from '@woocommerce/data';
+
+/**
+ * Internal dependencies
+ */
+import { getAdminSetting } from '~/utils/admin-settings';
+import { PaymentGatewayButton } from './payment-gateway-button';
+
+const assetUrl = getAdminSetting( 'wcAdminAssetUrl' );
+
+export const OfflinePaymentGateways = () => {
+	const { installedPaymentGateways, isResolving } = useSelect( ( select ) => {
+		return {
+			getPaymentGateway: select( PAYMENT_GATEWAYS_STORE_NAME )
+				.getPaymentGateway,
+			installedPaymentGateways: select(
+				PAYMENT_GATEWAYS_STORE_NAME
+			).getPaymentGateways(),
+			isResolving: select( PAYMENT_GATEWAYS_STORE_NAME ).isResolving(
+				'getPaymentGateways'
+			),
+		};
+	}, [] );
+
+	const [ isInitialized, setIsInitialized ] = useState< boolean >( false );
+
+	if (
+		! isInitialized &&
+		! isResolving &&
+		installedPaymentGateways.length > 0
+	) {
+		setIsInitialized( true );
```

Comment:
I might be missing something, but why do we need `isInitialized`? Looks like `isInitialized` is not used in other places. 🤔

Reply:
Good catch, I previously thought to manually handle loading while invalidate paymentGateway API, but ended up not  needing to. Removed it in 814e3c2